doc/api/tracing.md

```diff
@@ -2,10 +2,16 @@
 
 <!--introduced_in=v7.7.0-->
 
-> Stability: 1 - Experimental
```

Comment:
There is an ongoing [request](https://github.com/nodejs/diagnostics/issues/654) to migrate the current tracing event system to be perfetto based. The [last attempt](https://github.com/nodejs/node/pull/35900) also refactored the public APIs when perfetto is enabled.

Currently, the trace event module exports _legacy_ Chrome trace format. I think it would be prudent to defer marking this module as stable.

Reply:
Still in the middle of adopting the latest version of perfetto: https://github.com/legendecas/node/tree/perfetto/build. Hopefully I will split it into smaller PRs once I build a working solution.

Reply:
Removed the change on tracing.md!